lib/Conversion/TorchToTcp/Elementwise.cpp

```diff
@@ -105,6 +105,90 @@ class ConvertAtenMulOp : public OpConversionPattern<AtenMulTensorOp> {
   }
 };
 
+class ConvertAtenBatchNormOp : public OpConversionPattern<AtenBatchNormOp> {
+public:
+  using OpConversionPattern<AtenBatchNormOp>::OpConversionPattern;
+  using OpAdaptor = typename AtenBatchNormOp::Adaptor;
+
+  LogicalResult
+  matchAndRewrite(AtenBatchNormOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    Value input = adaptor.getInput();
+    RankedTensorType inputType = input.getType().dyn_cast<RankedTensorType>();
+
+    Value weight = adaptor.getWeight();
+    RankedTensorType weightType = weight.getType().dyn_cast<RankedTensorType>();
+
+    Value bias = adaptor.getBias();
+    RankedTensorType biasType = bias.getType().dyn_cast<RankedTensorType>();
+
+    Value runningMean = adaptor.getRunningMean();
+    RankedTensorType runningMeanType =
+        runningMean.getType().dyn_cast<RankedTensorType>();
+
+    Value runningVar = adaptor.getRunningVar();
+    RankedTensorType runningVarType =
+        runningVar.getType().dyn_cast<RankedTensorType>();
+
+    RankedTensorType resultType =
+        OpConversionPattern<AtenBatchNormOp>::getTypeConverter()
+            ->convertType(op.getType())
+            .template cast<RankedTensorType>();
```

Comment:
nit: I don't think the `.template` is needed

Reply:
Thanks, fixed!